llvm/test/Transforms/LoopVectorize/intrinsic.ll

```diff
@@ -506,7 +530,7 @@ for.body:                                         ; preds = %entry, %for.body
   %0 = load double, ptr %arrayidx, align 8
   %arrayidx1 = getelementptr inbounds double, ptr %z, i64 %indvars.iv
   %1 = load double, ptr %arrayidx, align 8
```

Comment:
```suggestion
  %1 = load double, ptr %arrayidx1, align 8
```
I think this was the intention. Currently the copysign call gets dropped by InstCombine.

Reply:
Excellent catch! Indeed, llvm.copysign.v4f64 should be matched by the fixed test.